Amazon.IonDotnet/Internals/Binary/RawBinaryReader.cs

```diff
@@ -135,6 +135,11 @@ private void ClearValue()
             _hasSymbolTableAnnotation = false;
         }
 
+        public void Dispose()
+        {
+            // Nothing to dispose for now
```

Comment:
Can we put that comment in the docs similar to how we have it for the driver's Result class:

`            /// <summary>
            /// Dispose the enumerator. No-op.
            /// </summary>
            public void Dispose()
            {
                return;
            }`

Reply:
done